test/images/sample-apiserver/Dockerfile

```diff
@@ -12,25 +12,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM k8s.gcr.io/kube-cross:v1.10.4-1 as build_k8s_1_10_sample_apiserver
+FROM us.gcr.io/k8s-artifacts-prod/build-image/kube-cross:v1.13.6-1 as build_k8s_1_17_sample_apiserver
 
 ENV GOPATH /go
 RUN mkdir -p ${GOPATH}/src ${GOPATH}/bin
 ENV PATH $GOPATH/bin:$PATH
 
-# The e2e aggregator test was originally added in #50347 and is designed to test ability to run a 1.7
-# sample-apiserver in newer releases. please see e2e test named "Should be able to support the 1.7 Sample
-# API Server using the current Aggregator"
-RUN go get -d k8s.io/sample-apiserver \
-    && cd ${GOPATH}/src/k8s.io/sample-apiserver \
-    && git checkout --track origin/release-1.10 \
-    && CGO_ENABLED=0 GOOS=linux GOARCH=BASEARCH go install .
+
+# The e2e aggregator test is designed to test ability to run sample-apiserver as an aggregated server.
+# see e2e test named "Should be able to support the 1.17 Sample API Server using the current Aggregator"
+
+# Build v1.17.0 to ensure the current release supports a prior version of the sample apiserver
```

Comment:
I did not expect this to change in 1.17... this was done in release-1.18 after 1.17 was released

Reply:
Dropped the sample-apiserver changes here.
[Jordan explained in Slack](https://kubernetes.slack.com/archives/C2C40FMNF/p1582771353152600?thread_ts=1582737004.145300&cid=C2C40FMNF) that the versions are purposely older here to test compatibility with aggregated API servers as far back as possible.